packages/angular/test/base/e2e/src/lazy/nested-outlet.spec.ts

```diff
@@ -27,5 +27,8 @@ describe('Nested Outlet', () => {
     cy.get('#goto-nested-page2').click();
   });
 
+  it('parentOutlet should be defined', () => {
```

Comment:
Suggestion: Add an "annotation" referencing the issue that lead to the creation of this test.